app/views/ajax/home.html.erb

```diff
@@ -39,7 +126,8 @@ document.addEventListener('DOMContentLoaded', (event) => {
       if(eventRequest.readyState === XMLHttpRequest.DONE) {
         if  (eventRequest.status === 201) {     // this is the expected return code for a POST to create a todo
           resultsDiv.textContent=eventRequest.responseText; // this will be the todo that is created
-        } else {
+
+          alert("ToDo List Successfully Created and Added to Database")
```

Comment:
This first of the two alerts looks misplaced.  It comes up if you do not get a 201, so the message that the todo was successfully created isn't correct.